### DISCUSSION TITLE
How would you set an "end" date to "now", where "now" means something hasn't ended yet.

### DISCUSSION THREAD

**Discussion by jerrac** [symfony/symfony#46950] [Q&A]
So, I think this is actually an interesting problem in general. Not just related to Symfony. 

My specific use case is that I want users to enter when they started something, and when they finished. But that something might still be ongoing, so they'd need a way to represent that. For example, you might keep track of your medications, and if you're still taking something, you put "now" in the "stopped taking on" date column. Or if you are putting your current job down on a resume, you'd put your end date as "current" or "now", or something like that.

How would you go about representing that kind of information in your app? A basic datetime field isn't going to do the trick.

My initial thought is to leave the end date field nullable, and then assume that the user means "now" when that field is blank. But I know I'd rather have an explicit way to enter "now" as a user... Especially if having an end date is important to business logic somehow. In that case I'd want to require the end date field to avoid bad data...

Anyone else have any thoughts?

**Comment by MatTheCat** [comment]
> My initial thought is to leave the end date field nullable, and then assume that the user means "now" when that field is blank. But I know I'd rather have an explicit way to enter "now" as a user.

Why would a user fill an end date if there is none? “I stopped now” isn’t the same thing than “I didn’t stop” at all. If there is no end date then `null` makes perfect sense.
